frame/ddc-metrics-offchain-worker/src/tests/test_runtime.rs

```diff
@@ -155,7 +155,7 @@ impl contracts::Config for Test {
     type DepositPerStorageItem = ();
     type RentFraction = ();
     type SurchargeReward = SurchargeReward;
-    type MaxDepth = MaxDepth;
+    type CallStack = [pallet_contracts::Frame<Self>; 31];
```

Comment:
Why exactly this logic?

Reply:
Because this value is used in the original runtime